### PR TITLE
lesson2 hw updated with completed example

### PR DESCRIPTION
**Pull request by nmohamed**

### CODE REVIEW

File: lesson2/README.md [hunk 1]
```diff
@@ -102,6 +102,6 @@ Draw on the whiteboards a diagram of how a currently existing social media app m
...
-3. The settings `fragment` will allow you to change the app's background color. Add three buttons that change the app's background color.
\ No newline at end of file
+1. This app will have two fragments and one activity. One fragments holds the to-do list, while the other fragment holds the settings page. The activity will have a button that lets you switch between fragments.
+2. The to-do list fragment should have five editable `textView`s. Add `onClick` listeners to them so that when you click on a to-do list item, an `alertDialog` will popup and ask for new text. The text the user inputs into the `alertDialog` will then replace the old text in the `textView`. Use a type of `layout` to organize the `textView`s. Don't worry if your app doesn't save the items you changed in your list when you switch fragments.
+3. The settings fragment will allow you to change the app's background color. Add three buttons that change the app's background color.
```
Comment (davidabrahams) [inline lesson2/README.md]: Maybe add:

For example, you could use a set of three bubble buttons that say "Red", "Blue", and "White".

File: lesson2/README.md [hunk 2]
```diff
@@ -102,6 +102,6 @@ Draw on the whiteboards a diagram of how a currently existing social media app m
...
-3. The settings `fragment` will allow you to change the app's background color. Add three buttons that change the app's background color.
\ No newline at end of file
+1. This app will have two fragments and one activity. One fragments holds the to-do list, while the other fragment holds the settings page. The activity will have a button that lets you switch between fragments.
+2. The to-do list fragment should have five editable `textView`s. Add `onClick` listeners to them so that when you click on a to-do list item, an `alertDialog` will popup and ask for new text. The text the user inputs into the `alertDialog` will then replace the old text in the `textView`. Use a type of `layout` to organize the `textView`s. Don't worry if your app doesn't save the items you changed in your list when you switch fragments.
```
Comment (davidabrahams) [inline lesson2/README.md:106]: Do they not know enough yet about having data persist across fragments? I forget. I'm pretty sure you're right. Just checking

File: lesson2/README.md [hunk 3]
```diff
@@ -50,7 +50,7 @@ However, with the Snapchat example, you could also have each swipeable screen be
 
 Alongside activities and fragments are other java files you can write, like classes, tests, or interfaces.
 
-[fragment]: https://cdn2.raywenderlich.com/wp-content/uploads/2015/10/android_fragments_d001_why_fragments.png "fragments are very cool, I promise"
+[fragment]: https://cdn2.raywenderlich.com/wp-content/uploads/2015/10/android_fragments_d001_why_fragments.png "i like fragments"
```
Comment (davidabrahams) [inline lesson2/README.md:53]: lol nice

File: lesson2/README.md [hunk 4]
```diff
@@ -88,16 +88,20 @@ When you want to run your app, press the green play icon in the top toolbar.  A
 3. Give it an id
 4. Go to the fragment java file
 5. Assign that button as a variable
-  * Note that the XML file does not make the button - it only describes the design of the button. You declare the button in the fragment/activity by using lookupbyid(), then assign the XML id to your button to describe the look of the button. XML is necessary if you want your button to actually appear on screen. This is true many other elements as well, not just buttons
-6. Make an onclicklistener
+  * Note that the XML file does not make the button - it only describes the design of the button. You declare the button in the fragment/activity by using `lookupbyid()`, then assign the XML id to your button to describe the look of the button. XML is necessary if you want your button to actually appear on screen. This is true many other elements as well, not just buttons
```
Comment (davidabrahams) [inline lesson2/README.md]: isnt it `findViewById`?

**Comment by davidabrahams** [comment]
Other than my few comments looks good 🚌  👍 💯

File: lesson2/README.md (hunk 4 above)
Comment (nmohamed) [reply lesson2/README.md]: that's right, idk why it says this woops

File: lesson2/README.md (hunk 2 above)
Comment (nmohamed) [reply lesson2/README.md:106]: I figured that if they do make data persist, then that's great, but for now to keep it simple and not worry about that

**Pull request merged**